app/components/pagination-pager/template.hbs

```diff
@@ -0,0 +1,52 @@
+<ul class="pagination">
+  {{#unless pager}}
+    <li class="previous {{if isFirstDisabled 'disabled'}}">
+      <a href="{{previousUrl}}" {{action 'previous'}}>{{{paginationPrevious}}}</a>
+    </li>
+
+    {{#each pages key="@index" as |page|}}
+      {{page-item disabled=disabled page=page selected=this.current seperator=seperator urlTemplate=urlTemplate pageSet='pageChanged'}}
+    {{/each}}
+
+    <li class="next {{if isLastDisabled 'disabled'}}">
+      <a href="{{nextUrl}}" {{action 'next'}}>{{{paginationNext}}}</a>
+    </li>
+  {{else}}
+    {{#if pagerFirst}}
+      <li class="{{if isFirstDisabled 'disabled'}}">
+        <a href="{{firstUrl}}" {{action 'first'}}>{{{pagerFirst}}}</a>
+      </li>
+    {{/if}}
+
+    <li class="{{if pagerSpread 'previous'}} {{if isFirstDisabled 'disabled'}}">
+      <a href="{{previousUrl}}" {{action 'previous'}}>{{{pagerPrevious}}}</a>
+    </li>
+
+    {{#if hasBlock}}
+      <li class="pager-inner">{{yield}}</li>
+    {{/if}}
+
+    <li class="{{if pagerSpread 'next'}} {{if isLastDisabled 'disabled'}}">
+      <a href="{{nextUrl}}" {{action 'next'}}>{{{pagerNext}}}</a>
+    </li>
+
+    {{#if pagerLast}}
+      <li class="last {{if isLastDisabled 'disabled'}}">
+        <a href="{{lastUrl}}" {{action 'last'}}>{{{pagerLast}}}</a>
+      </li>
+    {{/if}}
+  {{/unless}}
+</ul>
```

Comment:
The first part is exactly [this template](https://github.com/knownasilya/pagination-pager/blob/master/app/templates/components/pagination-pager.hbs), correct? Isn't there any way of including it instead of repeating it? If not, i'd suggest commenting that where this comes from and why it was extended.